logging/src/test/resources/log/logging-libs/java/logging/LoggerLibsClient.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package logging;
+
+import org.mule.runtime.api.exception.MuleException;
+import org.mule.runtime.core.api.event.CoreEvent;
+import org.mule.runtime.core.api.processor.Processor;
+
+import java.util.logging.Level;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LoggerLibsClient implements Processor {
```

Comment:
can this be added to the https://github.com/mulesoft/mule-test-artifacts as an operation and be reused from there? This way we also avoid adding new custom-processors

Reply:
The FakeMuleServer does [not](https://github.com/mulesoft/mule-integration-tests/blob/98fff96ed021d16318862c3aa3db7c2038aa540b/logging/src/test/java/org/mule/runtime/test/integration/logging/ConnectorLevelMessageDispatchingTestCase.java#L37) support extensions so this wouldn't be possible